setup.py

```diff
@@ -25,13 +25,11 @@
 Set the aeneas package up.
 """
 
-from setuptools import Extension
-from setuptools import setup
-import io
 import os
 import shutil
-import sys
 
+from setuptools import setup, Extension
```

Comment:
1. `setuptools isn't part of the python standard lib but the "de facto" standard. the official one is `distutil`https://docs.python.org/3/library/distutils.html. 

2. https://www.python.org/dev/peps/pep-0008/#imports says 
```It's okay to say this though:

from subprocess import Popen, PIPE
```